src/forum-tool-selector-settings/ForumToolSelectorSettings.jsx

```diff
@@ -0,0 +1,150 @@
+import React, { useState } from 'react';
+import { FormattedMessage } from '@edx/frontend-platform/i18n';
+import StudioApiService from '../data/services/StudioApiService';
+
+var selectedForumTool; // eslint-disable-line no-unused-vars
+var setSelectedFunctions = [];
+
+function resetSelected() {
+  for (var i = 0; i < setSelectedFunctions.length; i++) {
+    setSelectedFunctions[i](false);
+  }
+}
+
+function ForumCard(props) {
+  const [hovered, setHovered] = useState(false);
+  const toggleHover = () => setHovered(!hovered);
+  const [selected, setSelected] = useState(false);
+  setSelectedFunctions.push(setSelected);
+  const toggleSelected = () => {
+    if (selected === false) {
+      if (props.locked) return;
+      resetSelected();
+      setSelected(true);
+      selectedForumTool = props.id;
+    } else {
+      setSelected(false);
+      selectedForumTool = null;
+    }
+  }
```

Comment:
This state should not be in the component itself, this state is needed globally so it should be stored globally. The card should just take a param called "selected" and another called "onSelect" and leave the rest to the component using it.